### ISSUE TITLE
[Flaky Test] Networking Granular Checks: Services should function for service endpoints using hostNetwork

### ISSUE THREAD

**Issue by thejoycekung** [kubernetes/kubernetes#100890]
#### Which jobs are flaking:
ci-kubernetes-e2e-gce-cos-k8sbeta-default

#### Which test(s) are flaking:
Networking Granular Checks: Services should function for service endpoints using hostNetwork

#### Testgrid link:
https://testgrid.k8s.io/sig-release-1.21-blocking#gce-cos-k8sbeta-default

#### Reason for failure:
```
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/network/networking.go:489
Apr  7 00:02:38.725: Unexpected error:
    <*errors.errorString | 0xc000246240>: {
        s: "timed out waiting for the condition",
    }
    timed out waiting for the condition
occurred
/go/src/k8s.io/kubernetes/_output/dockerized/go/src/k8s.io/kubernetes/test/e2e/framework/network/utils.go:858
```

#### Anything else we need to know:
Triage looks like it's mostly flaking on this job only: 
https://storage.googleapis.com/k8s-gubernator/triage/index.html?test=Services%20should%20function%20for%20service%20endpoints%20using%20hostNetwork#2b29d5fd4b6ae220a82a

Examples
- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-cos-k8sbeta-default/1379578803428265984
- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-cos-k8sbeta-default/1379527874213580800
- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-cos-k8sbeta-default/1379314800881831936
- https://prow.k8s.io/view/gs/kubernetes-jenkins/logs/ci-kubernetes-e2e-gce-cos-k8sbeta-default/1379278309447176192

/sig network

**Comment by palnabarun** [comment]
/milestone v1.21

(marking in the current milestone until we get confirmation whether it is relevant to the release or not)

**Comment by palnabarun** [comment]
Slack Thread: https://kubernetes.slack.com/archives/CN0K3TE2C/p1617806533035800

**Comment by aojea** [comment]
This is fixed by #100893 , let's wait some runs to confirm before closing

**Comment by palnabarun** [comment]
/milestone clear

(since this has been deemed to be non release-blocking)

**Comment by aojea** [comment]
> This is fixed by #100893 , let's wait some runs to confirm before closing

/close
